### ISSUE TITLE
Deactivation

### ISSUE THREAD

**Issue by VTGCPM** [yikesinc/yikes-inc-easy-mailchimp-extender#739]
Hello, I have deactivated this plugin but the pop -up form is still showing on my site. How do I stop this from popping up?

**Comment by kpu23** [comment]
Hi @VTGCPM,

Is the popup coming from our premium popup plugin? What is controlling the popup?

Let me know,
Kevin.

**Comment by VTGCPM** [comment]
Hi Kevin,

I am not sure, I was not the one to set it up. I have deleted the plugin so I’m not sure how the form is still populating.

This is the form that pops up on the site.

[Inline image]

Kind Regards,

        [cid:image008.png@01D36DC3.78BED920] <http://www.choosegulfcoast.com/>


                Vanessa Torres
                Marketing Coordinator

Please make my day and share your feedback - Good or Bad!
[Good]<https://gatherkudos.com/gulfcoast/#ternary_choice_positive>



[Not bad]<https://gatherkudos.com/gulfcoast/#ternary_choice_neutral>



[Bad]<https://gatherkudos.com/gulfcoast/#ternary_choice_negative>

Click to rate your experience with Gulf Coast Property Management<https://gatherkudos.com/gulfcoast/>

**Comment by kpu23** [comment]
Hi Vanessa,

If the site you're referring to is https://www.choosegulfcoast.com/ then I believe the popup is coming from a plugin called "Optin Monster."

Does that sound right?

Let me know,
Kevin.

**Comment by VTGCPM** [comment]
Hi Kevin,

Wrong site,  the site this pop up was on is https://gcvacationrentals.com/

Kind Regards,

        [cid:image007.png@01D36DC4.48A09F20] <http://www.choosegulfcoast.com/>


                Vanessa Torres
                Marketing Coordinator

Please make my day and share your feedback - Good or Bad!
[Good]<https://gatherkudos.com/gulfcoast/#ternary_choice_positive>



[Not bad]<https://gatherkudos.com/gulfcoast/#ternary_choice_neutral>



[Bad]<https://gatherkudos.com/gulfcoast/#ternary_choice_negative>

Click to rate your experience with Gulf Coast Property Management<https://gatherkudos.com/gulfcoast/>

**Comment by kpu23** [comment]
Agh I see - the plugin controlling the popup is called `wbounce`.

**Comment by kpu23** [comment]
The `wbounce` modal is the one with our shortcode embedded in it. There's a second popup that looks like it's from MailChimp.com's embed code.

**Comment by VTGCPM** [comment]
Ahhh okay, I see it now.

The mailchimp one I just created.  It was the other one I was trying to take off the site.

So if I deactivate wbounce it will take off the popup correct?

Kind Regards,

        [cid:image007.png@01D36DC5.7FCAFA30] <http://www.choosegulfcoast.com/>


                Vanessa Torres
                Marketing Coordinator

Please make my day and share your feedback - Good or Bad!
[Good]<https://gatherkudos.com/gulfcoast/#ternary_choice_positive>



[Not bad]<https://gatherkudos.com/gulfcoast/#ternary_choice_neutral>



[Bad]<https://gatherkudos.com/gulfcoast/#ternary_choice_negative>

Click to rate your experience with Gulf Coast Property Management<https://gatherkudos.com/gulfcoast/>

**Comment by kpu23** [comment]
Yes deactivating or deleting the wbounce plugin will remove the popup.

**Comment by VTGCPM** [comment]
Awesome, thanks so much for your help!

Kind Regards,

        [cid:image007.png@01D36DC5.E52098E0] <http://www.choosegulfcoast.com/>


                Vanessa Torres
                Marketing Coordinator

Please make my day and share your feedback - Good or Bad!
[Good]<https://gatherkudos.com/gulfcoast/#ternary_choice_positive>



[Not bad]<https://gatherkudos.com/gulfcoast/#ternary_choice_neutral>



[Bad]<https://gatherkudos.com/gulfcoast/#ternary_choice_negative>

Click to rate your experience with Gulf Coast Property Management<https://gatherkudos.com/gulfcoast/>
